api/proto/attest/attest.proto

```diff
@@ -0,0 +1,49 @@
+// Copyright(c) 2020 Zededa, Inc.
+// All rights reserved.
+
+syntax = "proto3";
+
+option go_package  = "github.com/lf-edge/eve/api/go/certs";
+option java_package = "com.zededa.cloud.uservice.proto";
+
+// ZAttestNonceReq is request payload for POST /api/v1/edgeDevice/attestNonce
+// or /api/v2/edgeDevice/attestNonce
+// ZAttestNonceResp is response payload for POST /api/v1/edgeDevice/attestNonce
+// or /api/v2/edgeDevice/attestNonce
+// The message is assumed to be protected by a TLS session bound to the
+// device certificate for v1.
+// The message is assumed to be protected by signing envelope for v2.
+
+message ZAttestNonceReq {
+  string deviceUuid = 1;  //UUID assigned by the controller during register
+}
+message ZAttestNonceResp {
+  string nonce = 1;      //nonce to use in quote generation
+}
+
+// This is the request payload for POST /api/v1/edgeDevice/attestQuote
+// or /api/v2/edgeDevice/attestQuote
+// The message is assumed to be protected by a TLS session bound to the
+// device certificate for v1.
+// The message is assumed to be protected by signing envelope for v2.
+message ZAttestQuoteReq {
+  string deviceUuid = 1; //UUID assigned by the controller during register
+  string attestData = 2; //TPMS_ATTEST (Table 2:123) in https://trustedcomputinggroup.org/wp-content/uploads/TPM-Rev-2.0-Part-2-Structures-01.38.pdf
```

Comment:
Should this be bytes? Or is it guranteed to be a human-readable ASCII or UTF-8 string?

Reply:
changed it to bytes.